proto/compliance/compliance_info.proto

```diff
@@ -18,6 +18,17 @@ message ComplianceInfo {
   string reason = 9; 
   string owner = 10 [(cosmos_proto.scalar) = "cosmos.AddressString"];
   repeated ComplianceHistoryItem history = 11; 
+  string programTypeVersion = 12;
+  string CDCertificationID = 13;
+  string familyID = 14;
```

Comment:
Maybe use`familyId` to be consistent with other fields?

Reply:
Done! ✅